recipes/qt/5.x.x/conanfile.py

```diff
@@ -1391,14 +1391,18 @@ def _create_plugin(pluginname, libname, plugintype, requires):
                 if self.options.get_safe("qtwinextras"):
                     self.cpp_info.components["qtWinExtras"].system_libs.append("dwmapi")  # qtwinextras requires "DwmGetColorizationColor" which is in "dwmapi.lib" library
 
-
+            if is_apple_os(self):
+                self.cpp_info.components["qtCore"].frameworks.append("CoreServices" if self.settings.os == "Macos" else "MobileCoreServices")
+                self.cpp_info.components["qtNetwork"].frameworks.append("SystemConfiguration")
+                if self.options.with_gssapi:
+                    self.cpp_info.components["qtNetwork"].frameworks.append("GSS")
+                if not self.options.openssl: # with SecureTransport
+                    self.cpp_info.components["qtNetwork"].frameworks.append("Security")
             if self.settings.os == "Macos":
                 self.cpp_info.components["qtCore"].frameworks.append("IOKit")     # qtcore requires "_IORegistryEntryCreateCFProperty", "_IOServiceGetMatchingService" and much more which are in "IOKit" framework
                 self.cpp_info.components["qtCore"].frameworks.append("Cocoa")     # qtcore requires "_OBJC_CLASS_$_NSApplication" and more, which are in "Cocoa" framework
                 self.cpp_info.components["qtCore"].frameworks.append("Security")  # qtcore requires "_SecRequirementCreateWithString" and more, which are in "Security" framework
-                self.cpp_info.components["qtNetwork"].frameworks.append("SystemConfiguration")
-                if self.options.with_gssapi:
-                    self.cpp_info.components["qtNetwork"].frameworks.append("GSS")
+                self.cpp_info.components["PrintSupport"].system_libs.append("cups")
```

Comment:
```suggestion
```
Please put this after existing line 1142 (where PrintSupport module is declared). It would not make sense to definie some system_libs for a component which is not previously declared